resources/js/Jetstream/DialogModal.vue

```diff
@@ -1,48 +1,51 @@
 <template>
-    <modal :show="show" :max-width="maxWidth" :closeable="closeable" @close="close">
-        <div class="px-6 py-4">
-            <div class="text-lg">
-                <slot name="title">
-                </slot>
-            </div>
+  <modal :dusk="dusk" :show="show" :max-width="maxWidth" :closeable="closeable" @close="close">
```

Comment:
that one surprised me for a sec. was about tell you to put a better variable name till I realized this is a component